it/topics/scripting_languages/JavaScript/node_packages.md

```diff
@@ -0,0 +1,218 @@
+#  Node packages
+> Come gestire l'ambiente Node e i pacchetti Node
+
+Strumenti da riga di comando trattati qui:
+
+- [NPM](#npm)
+    - Installa le dipendenze di NodeJS.
+- [Yarn](#yarn)
+    - Installa le dipendenze di NodeJS nel modo moderno.
+- [NVM](#nvm)
+    - Gestisci più versioni di NodeJS.
+
+Vale la pena di sapere come utilizzare entrambi NPM _e_ Yarn. Alcuni progetti avranno preferenze rispetto a quello, ma spesso sono intercambiabili all'interno di un progetto. Yarn ha alcuni vantaggi oltre a NPM che vale la pena usare.
+
+
+## NPM
+
+Node viene fornito con il comando `npm`.
+
+- NPM home - [npmjs.com](https://npmjs.com/)
+- Home page dei documenti NPM - [docs.npmjs.com/](https://docs.npmjs.com/)
+- Messaggi
+    - [Un'introduzione al gestore pacchetti npm](https://flaviocopes.com/npm/) su blog.
+
+Vedi anche il mio [cheatheet NPM](https://github.com/MichaelCurrin/cheatsheets/blob/master/cheatsheets/package_managers/npm.md) se hai già familiarità con NPM.
+
+### Inizializza
+
+Prima di poter installare i pacchetti in un progetto, è necessario un file `package.json`.
+
+Se non ne hai uno, esegui questo:
+
+```sh
+$ npm init
+```
+
+### Directory dei moduli di Node
+
+L'esecuzione del comando install installerà i pacchetti di nodi in una directory chiamata `node_modules`.
+
+In un progetto, questo è alla radice del progetto.
+
+Per un'installazione globale, questo è qui sui sistemi Unix:
+
+```
+/usr/local/lib/node_modules
+```
+
+Puoi verificarlo con il comando `root`.
+
+```sh
+# Progetto
+npm root
+
+# Globale
+npm root -g
+```
+
+### Installazione del progetto
+
+Se installato in un progetto, aggiungi quella directory al tuo file `.gitignore`.
+
+Installa i pacchetti nel progetto corrente - se ometti un nome di pacchetto qui, allora deve esistere il file `package.json`.
+
+```Sh
+$ npm installa
```

Comment:
Code blocks should be left in original English otherwise they will not run.

Comments are fine to translate though